fee_allocator/accounting/fee_pipeline.py

```diff
@@ -129,11 +132,12 @@ def run_fees(
             Decimal(fee_constants["vebal_share_pct"]),
             aura_vebal_share=Decimal(aura_vebal_share),
         )
-        re_routed_incentives = re_route_incentives(_incentives, chain, reroute_config)
+        # re_routed_incentives = re_route_incentives(_incentives, chain, reroute_config)
```

Comment:
Why did we comment this out, if we ever used rerouting wouldn't we want to see it?

Reply:
moved back